src/controllers/social.js

```diff
@@ -0,0 +1,100 @@
+/* eslint-disable require-jsdoc */
+import User from '../services/user.service';
+import Helper from '../helpers/helper';
+import verifyUser from '../helpers/verification-email';
+
+let data;
+
+/**
+ *
+ *
+ * @class Social
+ */
+class Social {
+  // constructor(data = {}) {
```

Comment:
I think unused code should be removed

Reply:
Thanks Patrick, sorry forgot to remove commented-out code, let me work on these right away